includes/utilities.inc

```diff
@@ -460,11 +460,13 @@ function islandora_paged_content_ocr_combine(array $files, $out) {
   $success = TRUE;
   foreach ($files as $file) {
     $ocr = file_get_contents($file);
-    if ($ocr == FALSE) {
+    if ($ocr === FALSE) {
       $success = FALSE;
     }
     else {
-      $success = file_put_contents($out, $ocr, FILE_APPEND) == FALSE ? FALSE : $success;
+      if ($ocr) {
```

Comment:
I disagree with your example of the file containing a "0".  $ocr would be === "0" and that is not === FALSE, so then $success would be given the result of file_put_contents for that value of "0" with an append... and it would return the number of bytes written ... which would not be 0.

What might need to change is simply make the $success a boolean at the point of return like 
  return ($success === FALSE) ? FALSE : TRUE;